uSkyBlock-Core/src/main/po/cs.po

```diff
@@ -7,14 +7,15 @@ msgid ""
 msgstr ""
 "Project-Id-Version: \n"
 "Report-Msgid-Bugs-To: \n"
-"PO-Revision-Date: 2016-08-23 13:23+0100\n"
-"Last-Translator: martyHR\n"
+"POT-Creation-Date: \n"
+"PO-Revision-Date: 2016-09-28 08:20+0100\n"
+"Last-Translator: \n"
 "Language-Team: martyHR@LordSet <marty@hranice.cz>\n"
-"Language: cs\n"
+"Language: cz\n"
```

Comment:
I forgot to change that, I did ask the translator if they were the same language (he changed it so it didn't interfere with the other one). Ill fix this asap.